core/src/main/java/org/apache/accumulo/core/util/threads/ThreadPools.java

```diff
@@ -58,7 +82,9 @@ public static ExecutorService createExecutorService(AccumuloConfiguration conf,
       case MANAGER_RENAME_THREADS:
         return createFixedThreadPool(conf.getCount(p), "bulk move", false);
       case MANAGER_FATE_THREADPOOL_SIZE:
-        return createFixedThreadPool(conf.getCount(p), "Repo Runner", false);
+        ThreadPoolExecutor pool = createFixedThreadPool(conf.getCount(p), "Repo Runner", false);
+        makeResizeable(pool, conf, p);
```

Comment:
Seems like you could just have one method to create the resizeablePool.

Reply:
I figured it would just be easier to make one method that could modify any of the created thread pools rather than making more methods to create a thread pool or adding more parameters to all of the existing methods.